ci-operator/step-registry/openshift/microshift/infra/conformance-setup/openshift-microshift-infra-conformance-setup-commands.sh

```diff
@@ -6,9 +6,22 @@ curl https://raw.githubusercontent.com/openshift/release/master/ci-operator/step
 source /tmp/ci-functions.sh
 ci_script_prologue
 
+if "${SRC_FROM_GIT}"; then
+  mkdir -p /go/src/github.com/openshift
+  branch=$(echo ${JOB_SPEC} | jq -r '.refs.base_ref')
+  # MicroShift repo is recent enough to use main instead of master.
+  if [ "${branch}" == "master" ]; then
+    branch="main"
+  fi
+  git clone https://github.com/openshift/microshift -b $branch /go/src/github.com/openshift/microshift
+fi
+
 cp /go/src/github.com/openshift/microshift/origin/skip.txt "${SHARED_DIR}/conformance-skip.txt"
 cp "${SHARED_DIR}/conformance-skip.txt" "${ARTIFACT_DIR}/conformance-skip.txt"
```

Comment:
This is a little scary: https://github.com/openshift/microshift/blob/main/origin/skip.txt

Feels like two mechanisms for avoiding tests in microshift, one where we go disable it in origin and then this separate skips file. Do both need to exist and why?

Reply:
Ah, we started doing this as part of the origin fork we had and the strategy around it. Then we kept it because it has a fast loop over "I need to skip a test - It is finally skipped" than going through changes in origin. The idea is to shrink this until its not needed anymore as we start making changes in origin. Would that be enough?

Reply:
Think of this as an alternative implementation of [this](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml#L40-L46). It is the same concept but closer to the repo because we are working on it.